package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "start:dev": "webpack-dev-server --mode development --open --hot",
-    "start": "node server.js",
+    "start": "webpack-dev-server --mode development --open --hot",
```

Comment:
why is there is change @felixkiryowa

Reply:
It is required for other teammates to continue working on their tasks since one is not able to start a server using that command